### DISCUSSION TITLE
How to change default "export as SQL" script for Oracle

### DISCUSSION THREAD

**Discussion by tatod** [dbeaver/dbeaver#13521] [Q&A]
#### System information: 
- Operating system (distribution) and version: Ubuntu 20.04.2 LTS
- DBeaver version: 21.1.5.202108090656
- Additional extensions: n/a

#### Connection specification:
- Database name and version: Oracle Oracle Database 12c EE Extreme Perf Release 12.1.0.2.0 - 64bit Production
With the Partitioning, Real Application Clusters, Automatic Storage Management, OLAP,
Advanced Analytics and Real Application Testing options
- Driver name: Oracle JDBC driver 19.6.0.0.0
- Do you use tunnels or proxies (SSH, SOCKS, etc)? n/a

#### Describe the problem you're observing:
when need to export data from table as SQL there is parameter "Data rows per statement". In my case it was defaulted to 10. It has generated one insert with 10 values to be inserted. This does not work in oracle. Therefore I had to change it to 1 so I can get 1 insert per each value. Maybe it would make sense to default it to 1 for Oracle connections.

![2021-08-12_09-50-with_default_of_1row](https://user-images.githubusercontent.com/48986378/129159050-ce1489a4-3375-4c09-890e-d1325049d9b7.png)
![2021-08-12_09-50-with_default_of_10rows](https://user-images.githubusercontent.com/48986378/129159060-07b401ea-bbf7-40dd-b0fe-3d56312a9ce3.png)
![2021-08-12_09-49](https://user-images.githubusercontent.com/48986378/129159061-da731964-d7c0-4e9e-9b69-ac43ff67e37c.png)

#### Steps to reproduce, if exist:
right click table and select Export data and then select SQL and click Next. See parameter "Data rows per statement".

**Comment by LonwoLonwo** [comment]
Hello @tatod 

We have a special solution for Oracle.
Just choose "INSERT ALL" for "Upsert keyword" in Format settings.

![2021-08-12 13_23_06-DBeaver Enterprise 21 1 5 - ACTOR](https://user-images.githubusercontent.com/45152336/129181446-dc71d7fc-a9a2-4c44-a66e-0b8838ecc3bf.png)

So it will be generate something like this:
```sql
INSERT ALL
	INTO HR.ACTOR (ACTOR_ID,FIRST_NAME,LAST_NAME,LAST_UPDATE) VALUES (11,'ZERO','CAGE','2006-02-15 04:34:33')
	INTO HR.ACTOR (ACTOR_ID,FIRST_NAME,LAST_NAME,LAST_UPDATE) VALUES (16,'FRED','COSTNER','2006-02-15 04:34:33')
	INTO HR.ACTOR (ACTOR_ID,FIRST_NAME,LAST_NAME,LAST_UPDATE) VALUES (201,'NICK','WAHLBERG','2019-07-01 19:19:48')
SELECT 1 FROM DUAL;
```

**Comment by tatod** [comment]
I think I like INSERT ALL option :) 

Maybe then it would be better to default that one to avoid generating script which does not work but I leave this up to you to decide. I am fine to close this issue.
